### PR TITLE
fix(openrtb): Allow the minduration and maxduration fields be empty if there is no valid value.

### PR DESCRIPTION
**Pull request by DevKai**
# What
Allow the minduration and maxduration fields be empty if there is no valid value.
# Acceptance
The minduration and maxduration fields can be empty if there is no valid value.

### CODE REVIEW

File: openrtb/video.go [hunk 1]
```diff
@@ -4,8 +4,8 @@ package openrtb
 //easyjson:json
 type Video struct {
 	MimeTypes   []string `json:"mimes"`
-	MinDuration int      `json:"minduration"`
-	MaxDuration int      `json:"maxduration"`
+	MinDuration *int     `json:"minduration,omitempty"`
```
Comment (garukun) [inline openrtb/video.go]: Is it okay for us to assume 0 is also empty so that we can omit?

Comment (garukun) [inline openrtb/video.go]: Is it okay for us to assume 0 is also empty so that we can omit?

Comment (tiagozortea) [inline openrtb/video.go]: IMO anything <=0 should be omitted.

Comment (garukun) [inline openrtb/video.go]: I think <0 should be part of the Video validation.

Comment (DevKai) [inline openrtb/video.go]: @garukun. Currently, openrtb.BidRequest only validate openrtb.Application. We may need to create another pull request to deal with the BidRequest validation.

**Comment by garukun** [comment]
LGTM

**Pull request merged**